changelog.md

```diff
@@ -1,3 +1,140 @@
+<a name="6.2.0"></a>
+# 6.2.0 (2020-08-06)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.1.1...v6.2.0)
+
+We expect this release to ship in the DevTools of [Chrome 86](https://chromiumdash.appspot.com/schedule), and to PageSpeed
```

Comment:
look how much extra time we have in canary! 🎉  :D
![image](https://user-images.githubusercontent.com/2301202/89581779-31c65280-d7fd-11ea-9644-6c8fb1aca946.png)